src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -31,6 +31,8 @@ internal static class LibraryFlags
 
     internal static partial class Library
     {
+        internal static readonly List<FormulaType> AllowedListConvertToString = new List<FormulaType> { FormulaType.String, FormulaType.Number, FormulaType.DateTime, FormulaType.Date, FormulaType.Time, FormulaType.Boolean, FormulaType.Guid };
```

Comment:
>List [](http://example.com/codeflow?start=33&length=4)

Since this is shared state, it should be immutable type. Like IReadOnlyCollection. 


(I'm surprised that our Threading analysis tests doesn't flag this, since it's mutable shared state)

Reply:
updated this